boards/riscv/sparkfun_red_v_things_plus/board.cmake

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019 SiFive Inc.
```

Comment:
Wrong header?

Reply:
This file is just copy from `hifive1_revb` and identical to that.
(This board and `hifive1_revb` is very similar configuration.
 I want to keep fewer difference.)

Reply:
ack